src/gridstack.js

```diff
@@ -693,6 +693,12 @@
 
     opts = opts || {};
 
+    // if row property exists, replace minRow and maxRow
```

Comment:
move that down - feel weird to be first thing we check... like right after 715 where we set opts values.